toolkit/tools/imagegen/diskutils/diskutils.go

```diff
@@ -456,7 +456,7 @@ func InitializeSinglePartition(diskDevPath string, partitionNumber int, partitio
 			logger.Log.Debugf("Could not find partition path (%s). Checking other naming convention", testPartDevPath)
 		}
 		logger.Log.Warnf("Could not find any valid partition paths. Will retry up to %d times", totalAttempts)
-		err = fmt.Errorf("could not find partition to initialize in /dev")
+		err = fmt.Errorf("Could not find partition to initialize in /dev")
```

Comment:
This was actually correct - we've followed suggestion from the Go linters for VS Code to use small caps and no periods at the end for error messages.

Reply:
ah i see, thanks for pointing that out